ReadMe.md

```diff
@@ -1,3 +1,116 @@
 # MAGE SDK for JavaScript (ES5)
 
-This is a work-in-progress.
+This is the official MAGE SDK for the web. It is to be used in conjunction with MAGE server
```

Comment:
I don't think this paragraph is too relevant. If you want to have the chance to add a link to MAGE, would recommend:

Official [MAGE](https://www.npmjs.com/package/mage) client & SDK for web browsers.